src/main/scala/cognite/spark/v1/FlexibleDataModelCorePropertySyncRelation.scala

```diff
@@ -61,7 +61,7 @@ private[spark] class FlexibleDataModelCorePropertySyncRelation(
       HasData(List(viewRef))
     }
     val requestFilters: Seq[FilterDefinition] = (filters.map {
-      toNodeOrEdgeAttributeFilter(instanceType, _).toOption
+      toFilter(instanceType, _, None).toOption
```

Comment:
interesting that here we may have viewReference!=None, not sure if it'd be applicable to use it though

Reply:
I think that's what you were talking about, where we didn't want to pushdown filters on non-indexed columns

Reply:
let's add a comment that we don't pass viewRef to only do pushdown on `filters`? Though probably `filters` would only include what we want and we're safe either way? :thinking:

Reply:
Why would filter only include what we want?

Reply:
Ugh, tricky place indeed, before it was using `toNodeOrEdgeAttributeFilter` that only does pushdown on built-in attributes but incoming filters can be anything

Reply:
Though old code should fail with "Unsupported node or edge attribute filter" if filters contain anything else :thinking:

Reply:
That's true hmm, I wonder how the incoming filters are restricted then hmm...

Reply:
Should I leave a comment then or not? I guess somehow the sync calls is generated using only the correct filters, maybe its something in jetfire itself